README_ar.md

```diff
@@ -0,0 +1,602 @@
+
+3. اختيار الإطار الصحيح لكل جزء من عمر النموذج:
+    - تدريب نماذج حديثة في 3 أسطر من الكود.
+    - نقل نموذج واحد بين إطارات العمل TF2.0/PyTorch/JAX بحرية.
+    - اختيار الإطار الصحيح بسلاسة للتدريب، التقييم، والإنتاج.
+
+4. تخصيص نموذج أو مثال لاحتياجاتك بسهولة:
+    - نوفر أمثلة لكل هندسة لإعادة إنتاج النتائج التي نشرها مؤلفوها الأصليون.
+    - تتم تعرض دواخل النموذج بشكل متسق قدر الإمكان.
+    - يمكن استخدام ملفات النموذج بشكل مستقل عن المكتبة للتجارب السريعة.
+    
+## لماذا لايجب علي استخدام Transformers؟
+
+- هذه المكتبة ليست صندوق أدوات معياري من الكتل البنائية للشبكات العصبية. الكود الموجود في ملفات النموذج ليس معاد تنظيمه مع تجريدات إضافية عن عمد، حتى يتمكن الباحثون من التكرار بسرعة على كل من النماذج دون الغوص في تجريدات/ملفات إضافية.
+- واجهة برمجة التطبيقات للتدريب ليست مخصصة للعمل مع أي نموذج ولكنها مُحسنة للعمل مع النماذج التي توفرها المكتبة. لأجل حلقات التعلم الآلي العامة، يجب استخدام مكتبة أخرى (ربما، [Accelerate](https://huggingface.co/docs/accelerate)).
+- بينما نسعى لتقديم أكبر عدد ممكن من حالات الاستخدام، السكربتات في [مجلد الأمثلة](https://github.com/huggingface/transformers/tree/main/examples) لدينا هي مجرد ذلك: أمثلة. من المتوقع أنها لن تعمل مباشرةً على مشكلتك الخاصة وأنه سيتطلب منك تغيير بضعة أسطر من الكود لتكييفها مع احتياجاتك.
+
+## التثبيت
+
+### بواسطة pip
+
+تم اختبار هذا الريبو على +Python 3.8+، Flax 0.4.1+، PyTorch 1.11، و +TensorFlow 2.6.
+
+يجب تثبيت 🤗 Transformers في [بيئة افتراضية](https://docs.python.org/3/library/venv.html). إذا كنت غير مألوف ببيئات Python الافتراضية، راجع [دليل المستخدم](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/).
+
+أولًا، أنشئ بيئة افتراضية بإصدار Python الذي ستستخدمه وفعّلها.
+
+ثم، ستحتاج إلى تثبيت واحد على الأقل من Flax، PyTorch، أو TensorFlow.
+يرجى الرجوع إلى [صفحة تثبيت TensorFlow](https://www.tensorflow.org/install/)، [صفحة تثبيت PyTorch](https://pytorch.org/get-started/locally/#start-locally) و/أو [Flax](https://github.com/google/flax#quick-install) و [Jax](https://github.com/google/jax#installation) صفحات التثبيت لمعرفة الأمر المحدد لتثبيتها على منصتك.
+
+عند تثبيت أحد هذه الخلفيات، يمكن تثبيت 🤗 Transformers باستخدام pip على النحو التالي:
+
+```bash
+pip install transformers
+```
+
+إذا أردت التلاعب بالأمثلة أو كنت بحاجة إلى أحدث إصدار من الكود ولا يمكنك الانتظار لإصدار جديد، يجب عليك [تثبيت المكتبة من المصدر](https://huggingface.co/docs/transformers/installation#installing-from-source).
+
+### بواسطة conda
+
+يمكن تثبيت 🤗 Transformers باستخدام conda كالتالي:
+
+```shell script
+conda install conda-forge::transformers
+```
+
+> **_ملاحظة:_** تثبيت `transformers` من قناة `huggingface` أصبح مهجورًا.
+
+اتبع صفحات التثبيت لـ Flax، PyTorch أو TensorFlow لمعرفة كيفية تثبيتها باستخدام conda.
+
+> **_ملاحظة:_** على نظام التشغيل Windows، قد يُطلب منك تفعيل وضع المطور للاستفادة من التخزين المؤقت. إذا لم يكن هذا خيارًا لك، يرجى إعلامنا في [هذه المشكلة](https://github.com/huggingface/huggingface_hub/issues/1062).
```

Comment:
Same with link to the issue

Reply:
![image](https://github.com/huggingface/transformers/assets/71812454/154da4be-b46f-401f-afee-bff92550b0e2)